settings.gradle

```diff
@@ -32,7 +32,7 @@ include ':utilities'
 //include ':tools:benchmarksAnalyzer'
 include ':platformLibs'
 
-includeBuild 'shared'
+includeBuild 'version'
```

Comment:
How to make composite build with shared?

Reply:
Added the composite build. Now it's possible to specify path to a local shared repo using the corresponding property:
```
./gradlew dist -PsharedProjectPath=/path/to/shared
```